src/gui_common/CustomRichTextLabel.cs

```diff
@@ -323,12 +314,25 @@ private static string BuildTemplateForTag(string input, ThriveBbCode bbcode, Lis
             {
                 if (!InputMap.HasAction(input))
                 {
-                    GD.Print($"Input action: \"{input}\" doesn't exist, referenced in bbcode");
+                    GD.PrintErr($"Input action: \"{input}\" doesn't exist, referenced in bbcode");
                     break;
                 }
 
-                output = "[font=res://src/gui_common/fonts/BBCode-Image-VerticalCenterAlign-9.tres]" +
-                    $"[img=30]{KeyPromptHelper.GetPathForAction(input)}[/img][/font]";
+                var size = 30;
+                var ascent = 9;
+
+                if (pairs.TryGetValue("size", out string sizeInput))
+                    size = sizeInput.ToInt();
+
+                if (pairs.TryGetValue("ascent", out string ascentInput))
+                    ascent = ascentInput.ToInt();
+
+                var ascentFont = $"res://src/gui_common/fonts/BBCode-Image-VerticalCenterAlign-{ascent}.tres";
+
+                if (!ResourceLoader.Exists(ascentFont))
+                    GD.PrintErr($"Input Action: No ascent font found for {ascent}, fallback to using default");
```

Comment:
This doesn't seem to actually fallback to anything, just prints this error?

Reply:
`ascent` defaults to 9 in line 322.

Reply:
But on line 335, you use the `ascentFont` string anyway, which now has an invalid value?

Reply:
I just realized what you're saying, yes `ascent` needs to be reassigned.

Reply:
Nevermind, I end up doing another approach for this.